external/anomaly/tests/test_ote_training.py

```diff
@@ -0,0 +1,415 @@
+
+    @pytest.fixture
+    def test_case_fx(self, current_test_parameters_fx, params_factories_for_test_actions_fx):
+        """
+        This fixture returns the test case class OTEIntegrationTestCase that should be used for the current test.
+        Note that the cache from the test helper allows to store the instance of the class
+        between the tests.
+        If the main parameters used for this test are the same as the main parameters used for the previous test,
+        the instance of the test case class will be kept and re-used. It is helpful for tests that can
+        re-use the result of operations (model training, model optimization, etc) made for the previous tests,
+        if these operations are time-consuming.
+        If the main parameters used for this test differs w.r.t. the previous test, a new instance of
+        test case class will be created.
+        """
+        test_case = type(self).helper.get_test_case(current_test_parameters_fx,
+                                                    params_factories_for_test_actions_fx)
+        return test_case
+
+    # TODO(lbeynens): move to common fixtures
+    @pytest.fixture
+    def data_collector_fx(self, request) -> DataCollector:
+        setup = deepcopy(request.node.callspec.params)
+        setup['environment_name'] = os.environ.get('TT_ENVIRONMENT_NAME', 'no-env')
+        setup['test_type'] = os.environ.get('TT_TEST_TYPE', 'no-test-type') # TODO: get from e2e test type
+        setup['scenario'] = 'api' # TODO(lbeynens): get from a fixture!
+        setup['test'] = request.node.name
+        setup['subject'] = 'custom-anomaly-classification'
+        setup['project'] = 'ote'
+        if 'test_parameters' in setup:
+            assert isinstance(setup['test_parameters'], dict)
+            if 'dataset_name' not in setup:
+                setup['dataset_name'] = setup['test_parameters'].get('dataset_name')
+            if 'model_name' not in setup:
+                setup['model_name'] = setup['test_parameters'].get('model_name')
+            if 'test_stage' not in setup:
+                setup['test_stage'] = setup['test_parameters'].get('test_stage')
+            if 'usecase' not in setup:
+                setup['usecase'] = setup['test_parameters'].get('usecase')
+        logger.info(f'creating DataCollector: setup=\n{pformat(setup, width=140)}')
+        data_collector = DataCollector(name='TestOTEIntegration',
+                                       setup=setup)
+        with data_collector:
+            logger.info('data_collector is created')
+            yield data_collector
+        logger.info('data_collector is released')
+
+    @e2e_pytest_performance
+    def test(self,
+             test_parameters,
+             test_case_fx, data_collector_fx,
+             cur_test_expected_metrics_callback_fx):
+        if "nncf_graph" in test_parameters["test_stage"]:
+            pytest.xfail("The models has no a reference NNCF graph yet")
```

Comment:
Please, do not use XFail without Jira ticket number
Please, create a Jira ticket, assign it to Alexander Dokuchaev (@AlexanderDokuchaev, FYI!), write in 
a. description of the issue
b. the red bold phrase that the XFail in this test should be removed (and how it can be removed)

And after that, please,  point the ticket number to this XFail

Reply:
Done, added CVS-83365